DQMOffline/L1Trigger/interface/L1TStage2CaloLayer1Offline.h

```diff
@@ -0,0 +1,198 @@
+#ifndef L1TStage2CaloLayer1Offline_H
+#define L1TStage2CaloLayer1Offline_H
+
+//change from "One" to "Global"
```

Comment:
You can remove this kind of comments. Perhaps you can write a general comment saying that this is almost a duplicate of the online version of the L1TStage2CaloLayer1

Reply:
I can go change this. Will this require the code checks and tests be restarted?

Reply:
Yes, when you push the changes to your repository, the code-checks will start again automatically.
Please remove all comments similar to this one
Example:
`//#include "DQMServices/Core/interface/DQMOneEDAnalyzer.h" `
or
`//Elements removed from online version of module. [...]`